cif_pow.dic

```diff
@@ -3386,137 +3386,178 @@ save_PD_CALIB_XCOORD
     _definition.id                PD_CALIB_XCOORD
     _definition.scope             Category
     _definition.class             Loop
-    _definition.update            2023-06-10
+    _definition.update            2025-06-19
     _description.text
 ;
-    This section defines the parameters used for the calibration of the
-    X-coordinate axis of the instrument used directly or indirectly in the
-    interpretation of this data set; that is, 2\q, time-of-flight, or
-    position.
+    This category allows for the calibration of various X-coordinate axes
+    used in powder diffraction data sets by mapping a nominal X-coordinate
+    value to an actual X-coordinate value.
+
+    Supported X-coordinate axes include:
+    - 2theta
+    - d-spacing
+    - detected energy
+    - incident energy
+    - position
+    - Q (momentum transfer)
+    - time-of-flight
+    - wavelength
+    - detector (usually as channel number)
+
+    This mapping would usually be used to correct for systematic errors in
+    the X-coordinate axis, such as those caused by miscalibration of the
+    instrument, eg mapping measured 2\q values to corrected 2\q values, or to
+    provide a calibration for mapping a detector channel number to an actual
+    detected energy, d-spacing, or momentum transfer value.
 
-    Calibration is carried out be adding the offset value:
+    In general, any number of nominal-to-actual calibrations dould be
+    provided. Each line in the loop fixes the mapping from a nominal
+    value to an actual value. In essence, this is a machine-readable table
+    of values that would otherwise be given in a human-readable equation
+    in _pd_calibration.equation.
 
-        X-coord~calibrated~ = X-coord~meas~ + X-coord~offset~
+    Every datapoint in a diffractogram to be calibrated may not have an
+    equivalent value in the calibration table. In this case, an
+    interpolation must be made to obtain a value for each data point.
 
-    For cases where the offset value is not a constant, but rather varies
-    with X-coordinate, a set of offset values is supplied in a loop. In this
-    case, the value where the offset has been determined can be specified as
-    _pd_calib_xcoord.*_nominal. These values may need to be interpolated to
-    be applied to their respective diffractograms.
+    This category is not designed to correct for misaligned specimens - see
+    _pd_proc.* for the related X-coordinate data names.
 
-    Alternatively, if the offset has been determined at each measurement
-    point in the diffractogram, _pd_calib_xcoord.point_id may be used to
-    specify the offset on a per-point basis.
+    See also PD_CALIB_XCOORD_OVERALL.
 ;
     _name.category_id             PD_GROUP
     _name.object_id               PD_CALIB_XCOORD
 
     loop_
       _category_key.name
-         '_pd_calib_xcoord.diffractogram_id'
          '_pd_calib_xcoord.id'
+         '_pd_calib_xcoord.overall_id'
```

Comment:
I think `id` and `overall_id` as keys here works. It didn't work with `PD_PEAK` because every category that referred to a peak would need to have datanames that linked to peak/diffractogram/overall, and that would be a lot. `xcoord` is not referred to much if at all, so data name proliferation is less of a consideration.

Reply:
It also depends on the pd_instr, pd_instr_detector linkage. id and instr_id would work, as could get to the detectors from there. But yes, I get your point